tests/sbr-operator/tests/detect_only.go

```
@@ -200,9 +200,63 @@ var _ = Describe(
+
+				if agentDS.Status.DesiredNumberScheduled == 0 {
+					return fmt.Errorf("DaemonSet %s: no pods scheduled yet", dsName)
+				}
+
+				if agentDS.Status.CurrentNumberScheduled < agentDS.Status.DesiredNumberScheduled {
+					return fmt.Errorf("DaemonSet %s: %d/%d pods scheduled",
+						dsName, agentDS.Status.CurrentNumberScheduled, agentDS.Status.DesiredNumberScheduled)
+				}
+
+				listOpts := metav1.ListOptions{}
+				if agentDS.Spec.Selector != nil {
```

golangci-lint (2.13.2)

[error] missing whitespace above this line (no shared variables above if)

(wsl_v5)